src/consensus/aft/raft.h

```diff
@@ -1062,7 +1072,8 @@ namespace aft
 
       auto answer = (r.last_commit_term > last_commit_term) ||
```

Comment:
Comment on line `1070` may require updating too.

Reply:
I think it stands, we're just changing the definition of "up to date".